src/pages/stages.tsx

```diff
@@ -8,7 +8,7 @@ export type Stage = StagePageProps & {
 export const stages: Stage[] = [
```

Comment:
I think before we do that, we probably want to split this up? I agree that it makes sense to move some of this (e.g. how we link start and end events) into `lib` but the `PlanViewer` doesn't need everything. I don't think we want to move the `description` props into `lib` and we also don't need/want any reference to the routing moving either (and the `PlanViewer` doesn't need this).

I think this is an important consideration because we also want to make sure we're not including stuff we don't need in the CDN component.

Reply:
I have added a list of stages to lib/ with only the info needed by the PlanViewer component (no description and no route)